homeassistant/components/wled/light.py

```diff
@@ -83,20 +83,78 @@ async def async_setup_entry(
     update_segments()
 
 
-class WLEDLight(LightEntity, WLEDDeviceEntity):
-    """Defines a WLED light."""
+class WLEDMasterLight(LightEntity, WLEDDeviceEntity):
+    """Defines a WLED master light."""
+
+    def __init__(self, entry_id: str, coordinator: WLEDDataUpdateCoordinator):
+        """Initialize WLED master light."""
+        super().__init__(
+            entry_id=entry_id,
+            coordinator=coordinator,
+            name=f"{coordinator.data.info.name} Master",
+            icon="mdi:led-strip-variant",
+        )
+
+    @property
+    def unique_id(self) -> str:
+        """Return the unique ID for this sensor."""
+        return f"{self.coordinator.data.info.mac_address}"
```

Comment:
No need for the f-string since it's just that var.
```suggestion
        return self.coordinator.data.info.mac_address
```